source/creating_maps_charts/02_creating_advanced_charts/analyze.rst

```diff
@@ -1,41 +1,40 @@
 Analyzing with the Chart Builder
 ================================
 
-.. admonition:: Prerequisite
-   :class: important
+The Chart Builder tool allows you to visualize multiple datasets on a single customized chart.
 
-   Read how to :doc:`configure visualizations </publishing_data/07_configuring_visualizations/03_configuring_analyze_view/analyze>` for a dataset beforehand.
+It is usually made available by the domain administrator via a **Charts** link in the header menu. You can also access the Chart Builder through the following URL: ``http://<DOMAIN ADDRESS>/chart``.
 
-With the Chart Builder tool, you can visualize multiple datasets on a single customized chart. You can save your chart and easily share it, embed it, or use it as a widget.
+The Chart Builder is more flexible than **Analyze** tab. With the Chart Builder, you're not limited to one dataset. You can combine combining different sources from your domain and the Opendatasoft network.
 
-It is usually made available by the domain administrator in the top menu link under the codename `Charts`. The Map Builder can also be accessed using its URL: ``http://<DOMAIN ADDRESS>/chart``.
+After you create and save a chart, you can share it, embed it, or use it as a widget.
 
-.. admonition:: Note
-   :class: note
+.. admonition:: Prerequisite
+   :class: important
 
-   You can also build charts with `Analyze` tab on a dataset page, but these are limited to the current dataset. With the Chart Builder, you have more possibilities, combining different sources from your domain and from the Opendatasoft network.
+   Before you begin, see how to :doc:`configure visualizations </publishing_data/07_configuring_visualizations/03_configuring_analyze_view/analyze>` for a dataset.
 
 
 First layer
 -----------
 
-Click on ``Add a dataset to this layer`` to start importing data to your chart.
+Click on **Add a dataset to this layer** to start importing data to your chart.
 
 The list of available datasets is displayed: you can search or browse by category to find your datasets.
 As usual, you can preview and filter a dataset before importing it.
 
 As an example, let's take the public dataset "`Entreprises immatriculées en 2015`" from `Data Infogreffe` domain.
-You can ``Filter data`` to refine the data, but in our case let's just ``✓ Select`` the dataset.
+You can click **Filter data** to refine the data, but in our case, select the dataset.
 
-In the configuration, you can choose the appropriate type of graph. Select ``Column chart``
+In the configuration, you can choose the appropriate type of graph. Select **Column chart**.
 
-By default, Y-axis is set to *Count*, and the plot is a *Line*.
-You can choose the type of graph best suited to your data. Try selecting *Column chart* instead of *Line*.
+By default, Y-axis is set to **Count**, and the plot is a **Line**.
+You can choose the type of graph best suited to your data. Try selecting **Column chart** instead of **Line**.
```

Comment:
"Doc, I think we already followed that instruction in line 29"
"Oh my God, Marty, we must go back to delete line 29. If we follow the same instruction twice the universe will collapse"